lib/autoupdate.ps1

```diff
@@ -224,6 +224,11 @@ function get_hash_for_app([String] $app, $config, [String] $version, [String] $u
         $hashmode = 'sourceforge'
     }
 
+    if (!$hashfile_url -and $url -match "^.*mediafire.com\/\w+\/(?<random>\w+)\/(?<filename>.*)\/file(?:#\/.*)?$") {
+        $hashmode = 'mediafire'
```

Comment:
Using hashmode variable, which is used for `hash` extraction, to extract URL is pretty confusing

Reply:
Would not be better to just do `Invoke-RestMethod $url -match ...` here?